src/mlpack/core/optimizers/cmaes/test_function.hpp

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file test_function.hpp
```

Comment:
What is the reason behind this file, could we reuse the SGD test function, it looks like it's the same function?